Coding-Tracker/coding-tracker/coding-tracker/CodingSession.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace coding_tracker;
+
+public class CodingSession
+{   
+    public int Id { get; set; }
+    public string Session_description { get; set; }
+    public DateTime Date { get; set; }
+    public DateTime StartTime { get; set; }
+    public DateTime EndTime { get; set; }
+    public TimeSpan CalculateDuration
+    {
+        get
+        {
+            if (EndTime < StartTime)
+            {
+                return (EndTime.AddDays(1) - StartTime);
```

Comment:
This code is redundant as your validation logic doesn't allow `EndTime` to be less than `StartTime`.